devstack_vm/bin/utils.sh

```diff
@@ -244,3 +244,18 @@ rotate_log () {
 function timestamp(){
     echo `date -u +%H:%M:%S`
 }
+
+function cherry_pick() {
+    commit=$1
+    set +e
+    git cherry-pick $commit
+
+    if [ $? -ne 0 ]
+    then
+        echo "Ignoring failed git cherry-pick $commit"
+        git checkout --force
```

Comment:
As per a conversation we had with Mihai not long ago, you should use: git cherry-pick --abort (check git cherry-pick help)